billiard/pool.py

```diff
@@ -241,11 +241,12 @@ class Worker(Process):
 
     def __init__(self, inq, outq, synq=None, initializer=None, initargs=(),
                  maxtasks=None, sentinel=None, on_exit=None,
-                 sigprotection=True):
+                 sigprotection=True, max_mem_kb=None):
```

Comment:
can we use maxmemperchild here? Having kb in the name is useful, but does make the naming kind of awkward :(